index.html

```diff
@@ -2,6 +2,7 @@
 <DOCTYPE! html>
   <head>
      <title> first html and css project</title>
```

Comment:
please, It would be better if you insert <meta name="viewport" content="width=device-width, initial-scale=1.0">  viewport element in all your web pages after title element

Reply:
Kindly the same here explain `why` he would do that.